introduction.md

```diff
@@ -1,2 +1,8 @@
 I am king kuwanu man.
 Electives
+
+* JavaScript
+* C
+* Git
+* Code Schooling
+* Lazying
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?